website/docs/r/project_api_key.html.markdown

```diff
@@ -3,7 +3,7 @@ layout: "mongodbatlas"
 page_title: "MongoDB Atlas: project_api_key"
 sidebar_current: "docs-mongodbatlas-resource-project-api-key"
 description: |-
-    Provides a Project API Key resource.
+    Creates and assigns the specified Atlas Organization API Key to the specified Project. Users with the Project Owner role in the project associated with the API key can use the organization API key to access the resources.
 ---
 
 # Resource: mongodbatlas_project_api_key
```

Comment:
Do we want to add an example for project_assignment?

Reply:
Added create-api-key-assign-to-multiple-projects

Reply:
Thank you!